annotate.py

```diff
@@ -194,9 +197,14 @@ def annotate_dir(img_dir, dataset, subset, size1, size2,initials,filename) :
 
     global coords
     coords = [[], []]
-    
-    save_path = os.path.join(img_dir+dataset,subset+"_masks",subset,filename+".txt")
 
+    text_save_path = os.path.join(img_dir+"_texts")
+    mask_save_path = os.path.join(img_dir+"_masks")    
+    if not os.path.exists(text_save_path):
+        os.mkdir(text_save_path)
+    if not os.path.exists(mask_save_path):
+        os.mkdir(mask_save_path)
+    save_path = os.path.join(img_dir+"_texts",filename+".txt")
```

Comment:
See first comment above. This should go into the loop below, creating a new `.txt` file per image.